R/add_image.R

```diff
@@ -8,6 +8,7 @@
 #' @return A character object with an HTML fragment that can be placed inside
 #'   the message body wherever the image should appear.
 #' @examples
+#' \dontrun{
```

Comment:
What happened here?

Reply:
Let me check this again. Could be that the example didn't run for whatever reason.

Reply:
I removed this. In testing, it ran okay.

Reply:
On Travis we get an error:

```
> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
764> ### Name: add_image
765> ### Title: Create an HTML fragment for an embedded image
766> ### Aliases: add_image
767> 
768> ### ** Examples
769> 
770> # Create an HTML fragment that
771> # contains an image
772> img_file_path <-
773+   system.file(
774+     "img",
775+     "test_image.png",
776+     package = "blastula"
777+   )
778> 
779> img_file_html <-
780+   add_image(
781+     file = img_file_path
782+   )
783Warning in file(con, "rb") :
784  file("") only supports open = "w+" and open = "w+b": using the former
785Error in readBin(con = file, what = "raw", n = file.info(file)$size) : 
786  can only read from a binary connection
787Calls: add_image -> get_image_uri -> readBin
788Execution halted
```

Reply:
This seems specific to the build environment.

Reply:
Should I go back to using `dontrun{` and resolve to fix later?